ui/src/dashboards/components/CellEditorOverlay.tsx

```diff
@@ -326,11 +326,13 @@ class CellEditorOverlay extends Component<Props, State> {
     this.setState({isStaticLegend})
   }
 
-  private handleSetQuerySource = source => {
-    const queriesWorkingDraft = this.state.queriesWorkingDraft.map(q => ({
-      ..._.cloneDeep(q),
-      source,
-    }))
+  private handleSetQuerySource = (source: Source): void => {
+    const queriesWorkingDraft: QueryConfig[] = this.state.queriesWorkingDraft.map(
+      q => ({
+        ..._.cloneDeep(q),
+        source: source.links.self,
```

Comment:
this inconsistent namespacing feels confusing -- what do you think about renaming this field from `source` to `sourceLink`?

Reply:
Yea that makes sense